examples/set_colormaps.py

```diff
@@ -18,14 +18,17 @@
 with napari.util.app_context():
     v = napari.Viewer()
 
-    rlayer = v.add_image(rch)
+    rlayer = v.add_image(rch, name='red channel')
     rlayer.blending = 'additive'
-    rlayer.Colormap = 'red', red
-    glayer = v.add_image(gch)
+    rlayer.colormap = 'red', red
+    glayer = v.add_image(gch, name='green channel')
     glayer.blending = 'additive'
     glayer.colormap = green  # this will appear as [unnamed colormap]
-    blayer = v.add_image(bch)
+    blayer = v.add_image(bch, name='blue channel')
     blayer.blending = 'additive'
     blayer.colormap = {'blue': blue}
 
+    # set the blayer property widget to be expanded
+    blayer._qt_properties.setExpanded(True)
```

Comment:
Although I appreciate the utility of this (and wondered about doing this myself), I'm not sure we should have GUI code, or any private code at all, in the examples.

Reply:
Yes, this is maybe not so nice, a solution could be to have an expanded property on the layer directly that would trigger this change